### ISSUE TITLE
Using get5_endmatch command twice to end a match during map veto

### ISSUE THREAD

**Issue by yourmnbbn** [splewis/get5#540]
If making a feature request, you should delete all the pre-filled text here.
If reporting a bug, fill in the following sections:

### Expected behavior
When using get5_endmatch command during mapveto ,match will be ended after this command.  

### Actual behavior
When using get5_endmatch command during mapveto, Chat box will display message of both "An admin force ended the match." and "A captain left during the veto, pausing the veto",while actually the match is not ended.You have to use the get5_endmatch command again and this time the match is fully ended.  

Basically you have to use get5_endmatch twice to end a match during matpveto,I don't know if it was supposed to be like this or it is a bug.
  

### Steps to reproduce

- Plugin version: V0.7.2-dev Build #494 (2020-6-3 23:01:12)
- Sourcemod version: v1.11.0.6481  

- Steps to reproduce (please be specific):
1. Start a match through a match config,"skip_veto" set to 0
2. Use get5_endmatch during map veto.

**Comment by yourmnbbn** [comment]
I have found where the problem is and created a pull request.
